app/controllers/api/v1/posters_controller.rb

```diff
@@ -4,5 +4,28 @@ def index
         render json: Poster.all
     end
 
+    def show
+        poster = Poster.find(params[:id])
+        render json: poster
+    end
+
+    
+    def create
+        poster = Poster.create(poster_params)
+        render json: poster
+    end
+
+    def update
+        render json: Postere.update(params[:id], poster_params)    
```

Comment:
typo in the update method
I have some conditional logic I added in that method in the update post branch which we can discuss too!